ff2mpv.js

```diff
@@ -32,68 +32,106 @@ async function getOptions(id) {
 }
 
 async function submenuClicked(info) {
-  switch (info.parentMenuItemId) {
-    case "ff2mpv":
-      /* These should be mutually exclusive, but,
-         if they aren't, this is a reasonable priority.
-      */
-      url = info.linkUrl || info.srcUrl || info.selectionText || info.frameUrl;
-      if (url) {
-        const options = await getOptions(info.menuItemId);
-        ff2mpv(url, options);
-      }
-    break;
+  if (info.parentMenuItemId === 'ff2mpv' || info.menuItemId === 'ff2mpv') {
+    /* These should be mutually exclusive, but,
+       if they aren't, this is a reasonable priority.
+    */
+    const url = info.linkUrl || info.srcUrl || info.selectionText || info.frameUrl;
+    if (url) {
+      const options = await getOptions(info.menuItemId);
+      ff2mpv(url, options);
+    }
   }
 }
 
-function createProfile(profile) {
+let os = '';
```

Comment:
This mutable global state feels a little funny -- can't we call `getOS` twice or, even better, call it just once to set a global `TITLE` constant based on the right host?

Reply:
I've defined `TITLE` as a global variable. Let me know if it looks better know.